packages/lib/action/service.ts

```diff
@@ -122,10 +123,15 @@ export const createAction = async (data: TActionInput): Promise<TAction> => {
     },
   });
 
-  // revalidate cache
   revalidateTag(sessionId);
-  revalidateTag(getActionClassCacheTag(name, environmentId));
-  revalidateTag(getActionsCacheTag(environmentId));
+  actionClassCache.revalidate({
```

Comment:
We only need to revalidate actionClasses when a new actionClass is created. Can we check this somehow?

Reply:
but in this case it is an action created not actionClass

Reply:
and yeah we revalidate actionClasses by passing the environmentId

Reply:
I thought that we only need to revalidate the cache of all actionClasses when this `createAction` function is also creating a new action class. if the action class already exists and action classes are not changes (more likely case) we only need to revalidate the list of actions.

Reply:
oh I see. fixed